.gitattributes

```diff
@@ -0,0 +1 @@
+/src/tests/data/*/*.txt   -text
```

Comment:
It would be preferred to put .gitattributes to /src/tests/data, so it will not add more mess to the root folder.
Also, I would suggest to not apply it to all *.txt files but only affected ones, in case, for instance, we'll want to test whether cleartext signing works fine with both line endings/setups.

Reply:
@ni4 Done, please check it out